### PR TITLE
Add community page

### PR DESCRIPTION
**Pull request by tedivm**

### CODE REVIEW

File: source/contributed/community.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+* `#automation` has a group of users focused on 100% automation for their scripts.
+* `#client-abuse` discusses techniques for altering the game client by injecting html and javascript into the client through the in game console.
+* `#cpu-clinic` provides help for users who are having trouble figuring out why their bucket is always empty.
+* `#economy-clinic` helps users keep their economies flowing.
+* `#help` is the general purpose help channel.
+* `#loan` is for discussing development of the [League of Automated Nations site](http://www.leagueofautomatednations.com/).
+* `#operatingsystems` is filled with users who follow the "operating system model" of development as a way to scale and utilize their CPU.
+* `#politics` contains the majority of drama between players.
+* `#screepers` focuses on the third party development in the [Screepers github organization](https://github.com/screepers).
+* `#screepsplus` is the main support channel for the ScreepsPlus service.
```
Comment (ags131) [inline source/contributed/community.md]: I would like to request that ScreepsPlus get a link too please :)

Comment (tedivm) [inline source/contributed/community.md]: Done!

**Comment by artch** [comment]
Thanks, but I don't think this article has enough value to be added to official docs.

**Comment by tedivm** [comment]
I don't care about the credits if you don't think this was a high enough effort document, but I still think it would be useful to have this posted somewhere.

**Comment by starwar15432** [comment]
Doesn't care about the credits? Doesn't care about the credits?

**Comment by tedivm** [comment]
Yeah- I kind of knew this was a low effort article to begin with, but I also felt it was an important one. Showing the huge community, as well as the different facets of it, to prospective players makes them more likely to want to join (in my opinion at least), and giving a bunch of starting points for new players should help keep them from either missing out on something or being overwhelmed by all the options (particularly in slack, where there are a ton of channels).

So in this particularly case- low effort, high importance- I'm okay with skipping the subscription tokens.

**Pull request closed without merging**